.dockerignore

```diff
@@ -0,0 +1,7 @@
+coverage/
+docs/
+ghpages/
+test/
+tools/
```

Comment:
´This folder will be removed with https://github.com/telefonicaid/iotagent-json/pull/339 so I would suggest not to include it.

Reply:
Fixed - 5bb8375 - `tools/` removed.